helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -146,6 +146,17 @@ Define name for the Secret
 {{- end -}}
 {{- end -}}
 
+{{/*
+Define name for the etcd Secret
+*/}}
+{{- define "splunk-otel-collector.secret_etcd" -}}
```

Comment:
Nit: just for consistency with the established naming convention in helm

```suggestion
{{- define "splunk-otel-collector.etcdSecret" -}}
```

Reply:
Done